infrastructure/variables.tf

```diff
@@ -52,3 +52,11 @@ variable "business_area" {
 variable jenkins_AAD_objectId {}
 
 variable aks_subscription_id {}
+
+variable "enable_alerts" {
```

Comment:
since we have a few different alerts (overall) better to name it more specifically like `enable-task-deletion-failure-alert` or `enable-task-deletion-alert`